forum/qa-plugin/user-activity-log/user-activity-log-search.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+class user_activity_search{
```

Comment:
w CONTRIBUTING mamy info, że staramy się pisać kod w standardzie PSR-2. Wiem, że to miejscami niemożliwe przez same ograniczenia Q2A, ale np. zapisu klamer możemy próbować trzymać się jednego